src/Tizen.NUI/src/public/Color.cs

```diff
@@ -141,6 +141,28 @@ public Color(string hexColor) : this(Interop.Vector4.new_Vector4__SWIG_0(), true
             }
         }
 
+        /// <summary>
+        /// The conversion constructor from an System.Drawing.Color instance.
+        /// </summary>
+        /// <param name="color">System.Drawing.Color instance</param>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Color(global::System.Drawing.Color color) : this(Interop.Vector4.new_Vector4__SWIG_0(), true)
+        {
+            try
+            {
+                R = color.R / 255.0f;
+                G = color.G / 255.0f;
+                B = color.B / 255.0f;
+                A = color.A / 255.0f;
+            }
+            catch
+            {
+                throw new global::System.ArgumentException("Please check your System.Drawing.Color instance");
```

Comment:
could you check the following link?
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1065?view=vs-2019
even though this is not static constructor, if it throws exception in constructor, user could get some big confusion, so is there any other option to handle error case? (for example, giving a empty object or null ...)
I am not sure about this, so I am asking for your idea... ^^;